### PR TITLE
Cargo/Mining bomber jackets, minor tweaks to allowed items in cargo wintercoats

### PR DESCRIPTION
**Pull request by Christasmurf**
## What Does This PR Do
-Adds cargo and mining bomber jackets (Designs are up for discussion I am unsure how to feel)
-Puts mining bomber/winter coat in mining wardrobes
-Puts cargo bomber in cargodrobe
-Tweaks mining wintercoat to be able to hold kinetic accelerators, mining scanners and ore bags
-Tweaks cargo wintercoat to be able to hold clipboards and rapid crate senders

## Why It's Good For The Game
Extra variety for another department, extra functionality for existing items

## Images of changes
Cargo bomber
image
image
image
image

Mining bomber
image
image
image
image

## Testing
Booted test server, spawned as QM, checked all relevant sources of the jackets and all allowed items for functionality

## Changelog
:cl:
add: New cargo and mining bomber jackets (loadout and on station)
tweak: Cargo wintercoats can hold RCS and clipboard, mining wintercoats can hold KA, mining scanner and ore bags now
tweak: Mining wardrobes now contain wintercoats and bomber jackets
/:cl:

### CODE REVIEW

File: code/modules/clothing/suits/miscellaneous.dm [hunk 1]
```diff
@@ -830,6 +831,33 @@
...
 
+/obj/item/clothing/suit/jacket/cargobomber
+	name = "cargo bomber jacket"
+	desc = "A stylish jacket to keep you warm in the warehouse."
+	icon_state = "bombercargo"
+	item_state = "bombercargo"
+	ignore_suitadjust = 0
+	allowed = list(/obj/item/rcs, /obj/item/clipboard, /obj/item/flashlight, /obj/item/tank/internals/emergency_oxygen, /obj/item/toy, /obj/item/lighter, /obj/item/storage/fancy/cigarettes)
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
+	cold_protection = UPPER_TORSO|LOWER_TORSO|ARMS
```
Comment (hal9000PR) [inline code/modules/clothing/suits/miscellaneous.dm:842]:
```suggestion
	body_parts_covered = UPPER_TORSO | LOWER_TORSO | ARMS
	cold_protection = UPPER_TORSO | LOWER_TORSO | ARMS
```
I am aware we dont do this elsewhere, but spacing this kinda thing is good practice.

**Comment by Christasmurf** [comment]
> would you add a `/datum/gear/suit/mining_bomber_jacket` and `/datum/gear/suit/cargo_bomber_jacket` so that people can spawn with the item roundstart.

yeah will do where specifically do i add this?

**Comment by Christasmurf** [comment]
Reordered the loadout code so QM is placed first in the list in line with other departments having the roles in descending order

**Review by Sirryan2002 — APPROVED** [review]
code is fine for the most part, I don't know how loadout datums work but they look fine here as long as it was tested.

**Comment by Christasmurf** [comment]
> code is fine for the most part, I don't know how loadout datums work but they look fine here as long as it was tested.

Tested when I committed, spawned as QM with both items (one on suit slot, one in bag)

Not sure how to mark the requested changes as done so I suppose just need to wait on charlie to check and mark them

**Comment by Christasmurf** [comment]
Should be deconflicted

**Review by hal9000PR — APPROVED** [review]

**Pull request merged**